### ISSUE TITLE
build-package.sh fails to extract deb package with beta channel on Travis CI with Ubuntu Trusty

### ISSUE THREAD

**Issue by vzamanillo** [atom/ci#94]
### Description

CI tests with Travis and Ubuntu Trusty fails for atom beta channel.
```

jobs:
  include:
    - stage: test
      env: ATOM_CHANNEL=beta
```

**Expected behavior:**

Test passes.

**Actual behavior:**

```
$ curl -s -O https://raw.githubusercontent.com/atom/ci/master/build-package.sh
The command "curl -s -O https://raw.githubusercontent.com/atom/ci/master/build-package.sh" exited with 0.
0.00s$ chmod u+x build-package.sh
The command "chmod u+x build-package.sh" exited with 0.
0.84s$ ./build-package.sh
Downloading latest Atom release on the beta channel...
dpkg-deb: error: archive 'atom-amd64.deb' has premature member 'control.tar.xz' before 'control.tar.gz', giving up
Using Atom version:
./build-package.sh: 94: ./build-package.sh: /home/travis/atom/usr/bin/atom-beta: not found
Using APM version:
./build-package.sh: 96: ./build-package.sh: /home/travis/atom/usr/bin/apm-beta: not found
Downloading package dependencies...
./build-package.sh: 102: ./build-package.sh: /home/travis/atom/usr/bin/apm-beta: not found
Running specs...
./build-package.sh: 196: ./build-package.sh: /home/travis/atom/usr/bin/atom-beta: not found
The command "./build-package.sh" exited with 127.
```

**Reproduces how often:**

Always.

### Versions

atom-amd64.deb

```
dpkg-deb -I atom-amd64.deb 
 paquete Debian nuevo, versión 2.0.
 tamaño 108049684 bytes: archivo de control= 640 bytes.
     654 bytes,    12 líneas     control              
 Package: atom-beta
 Version: 1.39.0-beta3
 Depends: git, libgconf-2-4 (>= 3.2.5) | libgconf2-4, libgtk-3-0 (>= 3.9.10), libgcrypt11 | libgcrypt20, libnotify4, libxtst6, libnss3 (>= 2:3.22), python, gvfs-bin, xdg-utils, libx11-xcb1, libxss1, libasound2 (>= 1.0.16), libxkbfile1, libcurl3 | libcurl4, policykit-1
 Recommends: lsb-release
 Suggests: libsecret-1-0, gir1.2-gnomekeyring-1.0
 Section: devel
 Priority: optional
 Architecture: amd64
 Installed-Size: 631884
 Maintainer: GitHub <atom@github.com>
 Description: A hackable text editor for the 21st Century.
  Atom is a free and open source text editor that is modern, approachable, and hackable to the core.
```

Travis

`dist: trusty`

### Additional Information

I tryed to fix the issue based on the solution provided on https://github.com/travis-ci/travis-ci/issues/9361, upgrade to dpkg >= 1.17.5ubuntu5.8, which fixes https://bugs.launchpad.net/ubuntu/+source/dpkg/+bug/173062

```
install:
  - sudo apt-get install -qq -y dpkg
```

with no success

```
Downloading latest Atom release on the beta channel...

Using Atom version:

No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

The build has been terminate
```

I can confirm that changing the distro to `dist: xenial` (needs `libgconf-2-4` to run atom) builds the package and runs the tests without any problem so looks like the problem is not directly related to the build script.

**Comment by as-cii** [comment]
This issue started occurring after we transitioned from Ubuntu 14 to Ubuntu 18 in our build infrastructure. Initially, we thought it would have been fine to just switch all package builds to a newer Ubuntu distro, but during the last railcar release we realized that building on a newer distro would prevent people using older kernels from launching Atom at all. 

We have opened https://github.com/atom/atom/pull/19617 to go back to building Atom on Ubuntu 14 and we're actively working on releasing a hotfix on beta and stable. Once that's done, triggering a re-build should automatically fix the problem. Apologies for the oversight on this issue.

**Comment by jasonrudolph** [comment]
> We have opened [atom/atom#19617](https://github.com/atom/atom/pull/19617) to go back to building Atom on Ubuntu 14 and we're actively working on releasing a hotfix on beta and stable. Once that's done, triggering a re-build should automatically fix the problem. Apologies for the oversight on this problem.

atom/atom#19617 is merged, and we've published Atom 1.39.1 and 1.40.0-beta1 with that fix in place. I think that resolves this problem, so I'm going to close this issue. If you continue to experience this problem, please let us know. :bow:
